Storage/Storage/CoreData/CoreDataManager.swift

```diff
@@ -163,7 +154,6 @@ public final class CoreDataManager: StorageManagerType {
         viewContext.performAndWait {
             viewContext.reset()
             self.deleteAllStoredObjects(in: viewContext)
-            viewContext.saveIfNeeded()
```

Comment:
This is not needed as the view storage should only get merges from the persistent container automatically.